finetune/base.py

```diff
@@ -287,9 +287,7 @@ def _distribute_strategy(self, visible_gpus):
 
         resolved_gpus_string = ["/gpu:{}".format(gpu) for gpu in resolved_gpus]
         if len(resolved_gpus_string) == 1:
-            distribute_strategy = tf.distribute.OneDeviceStrategy(
-                resolved_gpus_string[0]
-            )
+            distribute_strategy = None
```

Comment:
With this change, TensorFlow respects setting `per_process_gpu_memory_fraction` in the config

Reply:
with this do we still observe the GPUs that are specified in the config?

Reply:
If I specify `CUDA_VISIBLE_DEVICES=0` for example, only GPU 0 is used. Is that what you meant?

Reply:
https://github.com/IndicoDataSolutions/finetune/blob/fd043734f87bdb0171ed4b0cb120d1375261664c/finetune/config.py#L201